aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -2426,3 +2454,30 @@ resource "aws_spot_fleet_request" "test" {
 }
 `, validUntil)
 }
+
+func testAccAWSSpotFleetRequestCapacityRebalance(rName string, validUntil string) string {
+	return testAccAWSSpotFleetRequestConfigBase(rName) + fmt.Sprintf(`
+resource "aws_spot_fleet_request" "test" {
+  iam_fleet_role                      = aws_iam_role.test.arn
+  spot_price                          = "0.05"
+  target_capacity                     = 2
+  valid_until                         = %[1]q
+  terminate_instances_with_expiration = true
+  wait_for_fulfillment                = true
+
+  spot_maintenance_strategies {
+    capacity_rebalance {
+      replacement_strategy = "launch"
+    }
+  }	
```

Comment:
Extra whitespace at EOL.

```suggestion
  }
```